charts/rancher-monitoring/101.1.0+up40.1.2/templates/exporters/core-dns/servicemonitor.yaml

```diff
@@ -31,6 +31,18 @@ spec:
 {{- if .Values.coreDns.serviceMonitor.metricRelabelings }}
     metricRelabelings:
 {{ tpl (toYaml .Values.coreDns.serviceMonitor.metricRelabelings | indent 4) . }}
+    {{ if .Values.global.cattle.clusterId }}
```

Comment:
same comment as above regarding moving `metrics-relabeling` outside the `if` statement.